shared/oae/css/oae.base.css

```diff
@@ -197,6 +197,40 @@ a:hover, .btn-link:hover, a:focus, .btn-link:focus {
     margin-top: 1px;
 }
 
+/* HR-like element that has a split with text in the middle of it */
```

Comment:
This might be worth turning into a component and giving a name that distinguishes it better from the regular `hr`